apis/io-engine/protobuf/v1/replica.proto

```diff
@@ -45,6 +47,7 @@ message Replica {
   bool is_snapshot = 11; // true: snapshot false: normal replica
   bool is_clone = 12; // true: clone replica from snapshot, false: normal replica
   optional string snapshot_uuid = 13; // snapshot uuid from which the clone replica is created
+  mayastor.v1.PoolType pooltype = 14; // type of the pool (eg: lvm, lvm)
```

Comment:
Typo in the comment, lvm mentioned twice.

Reply:
It's to emphasize lvm :D
I've also been wondering if instead of specifying type like this we should add oneof for the create requests, example:
```
oneof pooltype {
    LvmArgs lvm = 1;
    LvsArgs lvs = 2;
}
```
wdyt?

Reply:
Yes, it more sense. As We can have just one type for one pool.

Reply:
Sadly seems existing pool api already uses the type and not oneof, so probably better to do the same for consistency :(